src/Core/Components/AppBar/FluentAppBar.razor.cs

```diff
@@ -0,0 +1,115 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+using Microsoft.JSInterop;
+using System.Text.Json;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+
+public partial class FluentAppBar : FluentComponentBase
+{
+    private const string OVERFLOW_SELECTOR = ".appbar-item";
+    private readonly Dictionary<string, FluentAppBarItem> _apps = [];
+    private DotNetObjectReference<FluentAppBar>? _dotNetHelper = null;
+    private IJSObjectReference _jsModuleOverflow = default!;
+    private bool _showMoreItems = false;
+    private string? _searchTerm = string.Empty;
+    private IEnumerable<FluentAppBarItem> _searchResults = [];
+
+    /// <summary />
+    [Inject]
+    private IJSRuntime JSRuntime { get; set; } = default!;
+
+    [Parameter]
+    public RenderFragment? ChildContent { get; set; }
+
```

Comment:
Both Teams and new Outlook don't have this smaller version support. We can perhaps support this in a leter version but do not want to add it right now because of the extra work involved.